src/material-experimental/mdc-tabs/tab.ts

```diff
@@ -41,4 +43,16 @@ export class MatTab extends BaseMatTab {
   @ContentChild(MatTabLabel)
   override get templateLabel(): MatTabLabel { return this._templateLabel; }
   override set templateLabel(value: MatTabLabel) { this._setTemplateLabelInput(value); }
+
+  override ngOnChanges(changes: SimpleChanges): void {
+    super.ngOnChanges(changes);
+
+    // Triggering ChangeDetectorRef.markForCheck()
+    if (changes.hasOwnProperty('ariaLabel')
```

Comment:
Shouldn't this be in the base class?

Reply:
It looks like it works properly in the base class. I guess that is because different change detection strategy is used: **Default** in base and **OnPush** in mdc.
I've added that lines because discovered following tests fail for mdc only:
* https://github.com/angular/components/pull/23691/files#diff-4954b3ae980f3f5441b287520f08d854a09889b4679b4b73735813b1f5747f91R410
* https://github.com/angular/components/pull/23691/files#diff-4954b3ae980f3f5441b287520f08d854a09889b4679b4b73735813b1f5747f91R803
* https://github.com/angular/components/pull/23691/files#diff-4954b3ae980f3f5441b287520f08d854a09889b4679b4b73735813b1f5747f91R804
* https://github.com/angular/components/pull/23691/files#diff-4954b3ae980f3f5441b287520f08d854a09889b4679b4b73735813b1f5747f91R838
* https://github.com/angular/components/pull/23691/files#diff-4954b3ae980f3f5441b287520f08d854a09889b4679b4b73735813b1f5747f91R839

Reply:
If it's `Default` in the non-MDC version it should be the same in the MDC one. What likely happened was that we introduced the `Default` after the PR for the MDC one was opened.

Reply:
Yes, they should all be consistent.